environment.yml

```diff
@@ -0,0 +1,22 @@
+name: decomFL
+channels:
+  - pytorch
+  - conda-forge
+  - defaults
+dependencies:
+  - python=3.10.14
```

Comment:
there is no ~approx usage?

Reply:
we can pin 3.10, but I remember @RogerRogerUSC mentioned in wechat we are using 3.10.14

Reply:
either way are fine

Reply:
pin 3.10 instead of 3.10.14